### PR TITLE
Log reason for reconnect and underlying connection protocol

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

File: src/netlog/netlog-network.c [hunk 1]
```diff
@@ -167,16 +171,27 @@ int manager_network_connect_socket(Manager *m) {
...
+                default:
+                        assert_not_reached("Unknown protocol");
+        }
+
+        log_debug("Connecting to remote server: '%s/%s'", pretty, protocol);
 
         r = connect(m->socket, &m->address.sockaddr.sa, salen);
         if (r < 0 && errno != EINPROGRESS)
-                return log_error_errno(errno, "Failed to connect to remote server='%s': %m", pretty);
+                return log_error_errno(errno, "Failed to connect to remote server='%s/%s': %m", pretty, protocol);
```
Comment (ssahani) [inline src/netlog/netlog-network.c:190]: `protocol_to_string` should be suffice

File: src/netlog/netlog-network.c [hunk 2]
```diff
@@ -167,16 +171,27 @@ int manager_network_connect_socket(Manager *m) {
         if (r < 0)
                 return r;
 
-        log_debug("Connecting to remote server: '%s'", pretty);
+        switch (m->protocol) {
```
Comment (ssahani) [inline src/netlog/netlog-network.c:174]: we already doing it via `protocol_to_string`  no ?

**Review by ssahani — CHANGES_REQUESTED** [review]

File: src/netlog/netlog-network.c (hunk 1 above)
Comment (cgzones) [reply src/netlog/netlog-network.c:181]: True, thanks for the pointer; changed.

File: src/netlog/netlog-network.c (hunk 2 above)
Comment (cgzones) [reply src/netlog/netlog-network.c]: dito

**Pull request merged**